Bigtable/src/Connection/Grpc.php

```diff
@@ -293,7 +296,13 @@ public function testIamPermissions(array $args)
      */
     public function createTable(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $parent = $this->pluck('parent', $args);
+        return $this->send([$this->bigtableTableAdminClient, 'createTable'], [
+            $parent,
+            $this->pluck('tableId', $args),
+            new Table(),
```

Comment:
Just did a cursory look through and have a quick question: are there properties that can be configured when creating a table? It seems to be there may be, based off [the documentation](https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#google.bigtable.admin.v2.Table). Would we need to expose that to users?

Reply:
@dwsupplee yes there are optional properties that needs to be configured, we are not adding that right now. I will add todo annotations.

Reply:
Gotcha. sounds good.